images/source/centos-paas-sig-openshift-origin.repo

```diff
@@ -0,0 +1,5 @@
+[centos-paas-sig-openshift-origin]
+name = CentOS PaaS SIG Origin Repository
+baseurl = http://mirror.centos.org/centos/7/paas/x86_64/openshift-origin/
+gpgcheck = 0
+disable = origin* tuned* ansible* atomic* docker* kompose* openshift*
```

Comment:
It is "exclude = " not disable.  But if you are only wanting openvswitch, then that's simple  Get rid of the disable, and the exclude, and use includepkgs

includepkgs = openvswitch

That means "only use openvswitch from this repo"